.gitignore

```diff
@@ -11,6 +11,7 @@ res/version.rc
 res/version.txt
 res/julius.icns
 res/shell.html
+res/*assets
```

Comment:
Because augustus uses those folders and since I run both julius and Augustus forks as different local branches on the same folder, if I don't add that, git will want to add `res/assets` and `res/packed_assets` when I switch from augustus to Julius.